CHANGELOG.md

```diff
@@ -7,1218 +7,873 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [8.16.0] - 2025-09-16
-
+## [8.17.0] - 2025-09-30
 ### Added
+- Add support for user attribute profiles EA [#1166]
+- Add support for `user_attribute_profile_id` in `selfServiceProfiles` [#1166]
+- Add support for new language codes [#1159]
 
+## [8.16.0] - 2025-09-16
+### Added
 - Add support for token vault in `clients` and `resourceServers`. [#1158]
 - Add support for `async_approval` template in `emailTemplates`. [#1146]
 - Add support for `passkeys` in `prompts`. [#1161]
 
 ### Fixed
-
```

Comment:
I don't have a `kacl` installed on my system so the one from `node_modules` was used. Not sure why it removed all this whitespace given the in project one was used